Check-Substring.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <string.h>
+
```

Comment:
For input-  Enter first string: m*3
                  Enter second string: hem*3
expected output- m*3 is a substring of hem*3
your output - m*3 is not a substring of hem*3